settings/README.md

```diff
@@ -230,60 +238,61 @@ To react at a setting change by the user, you should use the signal
 `loadSetting` is called with the new settings.
 
 ```ts
-// src/index.ts#L47-L48
-
-// Listen for your plugin setting changes using Signal
-setting.changed.connect(loadSetting);
+// src/index.ts#L48-L51
 ```
 
-Finally, to demonstrate the programmatical change of a setting. A command to toggle
+Finally, to demonstrate the programmatical change of a setting, a command to toggle
 the `flag` setting is created.
 
 <!-- prettier-ignore-start -->
 ```ts
-// src/index.ts#L50-L61
-
-commands.addCommand(COMMAND_ID, {
-  label: 'Toggle flag setting',
-  isToggled: () => flag,
-  execute: () => {
-    // Programmatically change a setting
-    setting.set('flag', !flag).catch(reason => {
-      console.error(
-        `Something went wrong when setting flag.\n${reason}`
-      );
-    });
-  }
-});
+// src/index.ts#L56-L65
+
+label: 'Toggle Flag Setting',
+isToggled: () => flag,
+execute: () => {
+  // Programmatically change a setting
+  setting.set('flag', !flag).catch(reason => {
+    console.error(
+      `Something went wrong when setting flag.\n${reason}`
+    );
+  });
+}
```

Comment:
Are the first and last lines of the snippet missing?

Reply:
Done. Thx